community/faas/templates/subnetwork.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Create a subnetwork resource"""
+
+
+def generate_config(context):
+    """ Entry point for the deployment resources """
+
+    resource = [
+        {
+            'type': 'compute.v1.subnetwork',
+            'name': context.properties['name'],
+            'properties':
+                {
+                    'network': context.properties['network'],
+                    'ipCidrRange': context.properties['range'],
+                    'region': context.properties['region'],
+                    'privateIpGoogleAccess': True
```

Comment:
privateIpGoogleAccess should be configurable.

Reply:
Adding it...